### PR TITLE
ed: Tests modified to valid docker_test on optional NULL  value

### PR DESCRIPTION
**Pull request by kadhikari**

### CODE REVIEW

File: source/ed/tests/fusioparser_test.cpp [hunk 1]
```diff
@@ -221,10 +221,10 @@ BOOST_AUTO_TEST_CASE(parse_small_ntfs_dataset) {
     // headsign of vj and stop_times
     BOOST_REQUIRE_EQUAL(vj1->name.substr(0,6), "NULL");
 
-    BOOST_REQUIRE_EQUAL(vj1->stop_time_list[0]->headsign, "HS1");
```
Comment (antoine-de) [inline source/ed/tests/fusioparser_test.cpp:224]: why did you change this headsign ?

Comment (kadhikari) [inline source/ed/tests/fusioparser_test.cpp:224]: This was a headsign value that i had added in my last pull request. Infact it's better to have some empty, NULL and some other texts also. I will do that.

**Comment by kadhikari** [comment]
i modified some values in data and test file.

**Comment by antoine-de** [comment]
Jenkins, test this please

**Comment by pbougue** [comment]
Ok for me.
